clang/runtime/dpct-rt/include/dpct/memory.hpp

```diff
@@ -124,6 +124,13 @@ static inline bool is_device_ptr(T ptr) {
   }
   return false;
 }
+
```

Comment:
as it is public interface, please add comment for it.

Reply:
Updated